code/game/objects/structures/dropship_ammo.dm

```diff
@@ -140,12 +139,7 @@
 	name = "high-velocity 30mm ammo crate"
 	icon_state = "30mm_crate_hv"
 	desc = "A crate full of 30mm high-velocity bullets used on the dropship heavy guns."
-	travelling_time = 6 SECONDS
-	ammo_count = 400
-	max_ammo_count = 400
-	ammo_used_per_firing = 40
-	bullet_spread_range = 5
-	point_cost = 150
```

Comment:
hm ?

Reply:
Deleting point cost was an accident but it's a subtype of regular 30mm, so it inherits everything else from the 30mm